packages/web-app-files/src/App.vue

```diff
@@ -110,8 +110,6 @@ main {
 }
 
 .files-list-wrapper {
```

Comment:
i think height and min-height still needed

Reply:
I've set them on the wrapping div I think